core/arch/arm/kernel/link.mk

```diff
@@ -28,7 +28,8 @@ link-script-cppflags := -DASM=1 \
 entries-unpaged += thread_init_vbar
 entries-unpaged += sm_init
 entries-unpaged += core_init_mmu_regs
-entries-unpaged += stack_tmp_top
+entries-unpaged += stack_tmp
```

Comment:
I'd say the rule is to try to get rid of as much as possible of `entries-unpaged` as `KEEP_PAGER(x)` can be used directly in the source code instead of some other not obviously related file.